core/src/main/java/org/elasticsearch/search/fetch/subphase/DocValueFieldsFetchSubPhase.java

```diff
@@ -37,8 +41,44 @@
  */
 public final class DocValueFieldsFetchSubPhase implements FetchSubPhase {
 
+//    @Override
+//    public void hitExecute(SearchContext context, HitContext hitContext) throws IOException {
```

Comment:
Yep, only leaving it here until I ensure the tests pass so I have a reference of the old code